source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,274 @@
+===========
+Search Text
+===========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you will learn how to run a **text search** in MongoDB.
+
+You can use a text search to search documents for a term or a phrase. A
+term is a sequence of characters that excludes whitespace characters. A
+phrase is a sequence of terms that with any number of whitespace
+characters. You can use this for searching through unstructured text,
+which is text without a predefined format such as resumes and
+course catalogs. 
```

Comment:
Perhaps it's the way it is worded, but this makes me wonder what is considered structured text and why you can't search it with text search. Could you either explain it in more detail or omit the term if possible to avoid confusion?